### DISCUSSION TITLE
How to disable failed errors on start up?

### DISCUSSION THREAD

**Discussion by utuxedokamen** [atom/atom#22703] [Q&A]
Every time when I open program, it shows error about failed things. How can I disable that?

thanks in advance.

**Comment by icecream17** [comment]
Can you screenshot the error that happens?

**Reply by utuxedokamen** [reply]
It goes like this and pops up every time when I open a new window.

Failed to load snippets from 'C:\Users\user.atom\packages\atom-bootstrap4\snippets\components-button_group.cson'

C:\Users\user\.atom\packages\atom-bootstrap4\snippets\components-button_group.cson: Duplicate key 'Button group - Bootstrap 4'
Failed to load snippets from 'C:\Users\user.atom\packages\atom-bootstrap4\snippets\components-modals.cson'

C:\Users\user\.atom\packages\atom-bootstrap4\snippets\components-modals.cson: Duplicate key 'Modal content based- Bootstrap 4'
Failed to load snippets from 'C:\Users\user.atom\packages\atom-bootstrap4\snippets\components-pagination.cson'

C:\Users\user\.atom\packages\atom-bootstrap4\snippets\components-pagination.cson: Duplicate key 'Pagination - Bootstrap 4'
Failed to load snippets from 'C:\Users\user.atom\packages\atom-bootstrap4\snippets\components-progress.cson'

C:\Users\user\.atom\packages\atom-bootstrap4\snippets\components-progress.cson: Duplicate key 'Progress - Bootstrap 4'
Failed to load snippets from 'C:\Users\user.atom\packages\atom-bootstrap4\snippets\components-cards.cson'

C:\Users\user\.atom\packages\atom-bootstrap4\snippets\components-cards.cson: Duplicate key 'Cards groups - Bootstrap 4'

**Reply by icecream17** [reply]
Hmm, maybe try removing the package `atom-bootstrap4`?

cmd
```cmd
cd C:\Users\user.atom\packages
rm -rf atom-bootstrap4
```
